platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/eventhandling/ConsensusRoundHandlerTests.java

```diff
@@ -129,7 +129,7 @@ void normalOperation() throws InterruptedException {
         verify(swirldStateManager).handleConsensusRound(consensusRound);
         verify(swirldStateManager, never()).savedStateInFreezePeriod();
         verify(platformState)
-                .setRunningEventHash(
+                .setLegacyRunningEventHash(
```

Comment:
The `ConsensusRoundHandlerTests` should probably be expanded to cover the new running hash functionality